crates/node_binding/binding.d.ts

```diff
@@ -1319,6 +1319,7 @@ export interface RawLightningCssBrowsers {
 }
 
 export interface RawLightningCssMinimizerOptions {
+  Minify?: boolean
```

Comment:
Should be `Minify`, have you updated this file manually? it is generated automatically.